### ISSUE TITLE
Why in schematic - operation amplifier is - TL074?

### ISSUE THREAD

**Issue by Ildaron** [upsidedownlabs/BioAmp-EXG-Pill#28]
looks like some misprint

**Comment by lorforlinux** [comment]
It's not a misprint, TL074H/TL084H can be used.

**Comment by sarkrui** [comment]
Hi @lorforlinux  i am considering to remake one using smaller SMD packaging OpAmp chips. Would you recommend ADA4691-4ACPZ-R7 for example? plus, would anything wrong (any disadvantages) if the system power supply is 3.3v, i am thinking to use nRF52 chips as the MCU. Thank you!

**Comment by AdamM68** [comment]
nrf52 :)  
waiting for !

**Comment by lorforlinux** [comment]
@sarkrui Yes, that amplifier looks like a good choice. make sure nrf52 ADC can handle full 0v - 3.3v analog input, esp32 can only record 2.1v/2.5v/3.1v depending on the type of esp32 you are using, and for that type of ADC, you have to update the reference power supply voltage divider circuit to create Ref voltage = (ADC max voltage readable) / 2. I think it would be better if you create another issue if you have any other questions.

@AdamM68 We have created a new board with 3ch EXG input capability, we have tested it for ECG, EMG, EOG, and EEG. It works very well for all biopotential signals. If you are interested in getting pre-release units for your projects you can email us at contact [at] upsidedownlabs.tech

![image](https://github.com/user-attachments/assets/e9b80059-ac36-456d-af73-813c8fe0d354)

**Comment by DanielFasel** [comment]
What made you settle for the TL074H? I'm looking at amplifiers to build 64 channel EMG setup and a bit lost with all the options..

**Comment by rizmyabdulla** [comment]
> [@AdamM68](https://github.com/AdamM68) We have created a new board with 3ch EXG input capability, we have tested it for ECG, EMG, EOG, and EEG. It works very well for all biopotential signals. If you are interested in getting pre-release units for your projects you can email us at contact [at] upsidedownlabs.tech
> 
> ![image](https://github.com/user-attachments/assets/e9b80059-ac36-456d-af73-813c8fe0d354)

Hello, Can you share the schema for this new board
